db-client-java/src/test/java/com/eventstore/dbclient/ParseValidConnectionStringTests.java

```diff
@@ -91,7 +91,11 @@ public static Collection validConnectionStrings() {
                 {
                         "esdb://host?MaxDiscoverAttempts=200&discoveryinterval=1000&GOSSIPTIMEOUT=1&nOdEpReFeReNcE=leader&TLS=false&TlsVerifyCert=false&THROWOnAppendFailure=false",
                         "{\"dnsDiscover\":false,\"maxDiscoverAttempts\":200,\"discoveryInterval\":1000,\"gossipTimeout\":1,\"nodePreference\":\"leader\",\"tls\":false,\"tlsVerifyCert\":false,\"throwOnAppendFailure\":false,\"hosts\":[{\"address\":\"host\",\"port\":2113}]}"
-                }
+                },
+                {
+                        "esdb://localhost?keepAliveTimeout=20&keepAliveInterval=10",
```

Comment:
**CONCERN:** Can we add more test scenarios? E.g.
- when values equal to `-1` are provided,
- when values smaller than `-1` are provided,
- plus some permutations with valid and invalid values.

Reply:
Done